src/core/Context.py

```diff
@@ -99,33 +120,45 @@ async def success(self, message, delete_after=None, **kwargs):
                 delete_after=delete_after,
                 **kwargs,
             )
+        return None
 
-    async def simple(self, message, delete_after=None, **kwargs):
+    async def simple(
+        self, message: str, delete_after: Union[int, float] = None, **kwargs: Any
+    ) -> Optional[discord.Message]:
         with suppress(discord.HTTPException):
             image = kwargs.pop("image", None)
             footer = kwargs.pop("footer", None)
 
+            embed = discord.Embed(description=message, color=self.bot.color)
+            if image:
```

Comment:
since dpy has depreciated `discord.Embed.Empty`, I don't think there's any harm in explicitly setting image or footer text as None